cclib/scripts/ccframe.py

```diff
@@ -63,7 +63,12 @@ def main():
                         help=('name of sheet which will contain DataFrame, if '
                               'writing to an Excel file, or identifier for '
                               'the group in HDFStore, if writing a HDF file'))
+    parser.add_argument('-f', '--force', action='store_true',
+                        help=('overwrite output file in case it already exists'))
     args = parser.parse_args()
+    if args.output is not None and not args.force and os.path.exists(args.output):
+        parser.exit(1, f'failure: exiting to avoid overwriting existing file "{args.output}"\n')
```

Comment:
There's a syntax error on this line: https://travis-ci.org/github/cclib/cclib/jobs/661763079#L1077

It might be prudent to add a unit test covering this new code.

Reply:
Oops sorry, Python 2.7 doesn't support f-strings.